.github/workflows/release.yaml

```diff
@@ -0,0 +1,151 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: The version you intend to release (eg x.y.z)
+  pull_request:
+    types: [ closed ]
+  push:
+    branches:
+      - 'release/**'
+    tags:
+      - v*
+
+env:
+  VERSION: ${{ github.event.inputs.version }}
+  RELEASE_MINISIGN_PRIVATE_KEY: ${{ secrets.RELEASE_PRIVATE_KEY }}
+  RELEASE_MINISIGN_PRIVATE_KEY_PASSWORD: ${{ secrets.RELEASE_PRIVATE_KEY_PASSWORD }}
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.version != '' }}
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v1
+        with:
+          app_id: 251311
```

Comment:
what is app_id?

Reply:
uid of the github app installed on the repo